src/components/RightSidebar/RightSidebar.vue

```diff
@@ -141,6 +142,13 @@ export default {
 		opened() {
 			return !!this.token && !this.isInLobby && this.show
 		},
+		showLargeBox() {
```

Comment:
Maybe we can come up with a better name here? Like showSidebarPlaceholder or something similar.
Also there should be some documentation on top of this computed that describes what this does since it's a bit complicated to figure out